modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util/navigate.es.js

```diff
@@ -32,7 +32,25 @@ export default function (url, listeners) {
 			});
 		}
 	}
-	else {
+	else if (_isValidURL(url)) {
 		window.location.href = url;
 	}
 }
+
+function _isValidURL(url) {
+	var url_object;
```

Comment:
Our convention is to use _camelCase_ for variable names. Also, we should use `let` here instead of the old rusty `var`.

Reply:
I renamed the variable to `urlObject` and make it a `let`.